template/{{.input_root_dir}}/.github/workflows/{{.input_project_name}}-run-tests.yml.tmpl

```diff
@@ -16,8 +18,6 @@ env:
   ARM_CLIENT_SECRET: {{`${{ secrets.STAGING_AZURE_SP_CLIENT_SECRET }}`}}
   {{- end }}
 
-concurrency: {{template `project_name` .}}-feature-training-integration-test-staging
```

Comment:
Why do we remove the concurrency? Two PRs of the same project could run integration test in the same staging workspace and cause failures.

Reply:
Oh I thought concurrency is to prevent different jobs in the same workflow from running together at the same time

Reply:
Nvm looks like I misunderstood it, re-added it

Reply:
👍